src/System.Management.Automation/engine/ArgumentTypeConverterAttribute.cs

```diff
@@ -152,11 +152,6 @@ internal object Transform(EngineIntrinsics engineIntrinsics, object inputData, b
                         if (_convertTypes[i] == typeof(ActionPreference))
```

Comment:
@KirkMunro, your last commit had 86 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/engine/Api/TypeInference.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 147
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Host/Logging.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 9
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Classes/ProtectedAccess.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Classes/ProtectedAccess.Tests.ps1: line 55
```
Error occurred in test script '/Users/vsts/agent/2.154.3/work/1/s/test/powershell/Language/Classes/Scripting.Classes.Attributes.Tests.ps1'
```powershell
Index was outside the bounds of the array.
at <ScriptBlock>, /Users/vsts/agent/2.154.3/work/1/a/bins/publish/Modules/Pester/4.8.0/Pester.psm1: line 1097
```